# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/librsvg/missing-requires-in-pc.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./librsvg-2.0.pc.in.orig	2004-12-04 19:39:28.818414896 +0100
+++ ./librsvg-2.0.pc.in	2004-12-04 19:44:15.204877536 +0100
@@ -6,6 +6,6 @@
 Name: librsvg
 Description: library that renders svg files
 Version: @VERSION@
-Requires: glib-2.0 gdk-pixbuf-2.0
+Requires: glib-2.0 gdk-pixbuf-2.0 libgsf-1 libcroco-0.6 gnome-vfs-2.0
 Libs: -L${libdir} -lrsvg-2
 Cflags: -I${includedir}/librsvg-2
